.github/workflows/ci.yml

```diff
@@ -40,13 +40,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.12"]
+        python-version: ["3.9", "3.12"]
         runs-on: [ubuntu-latest, macos-latest, windows-latest]
-
-        include:
```

Comment:
Slicer runs Python 3.9, so I raised the minimum version everywhere to 3.9. I suppose we _could_ leave the minimum version 3.8, and just _add_ 3.9 to the test matrix... but I'm not sure how important 3.8 support is, and it means we lose out on https://docs.python.org/3/whatsnew/3.9.html

`removeprefix`, `removesuffix`, builtin generics (eg `list[str]`), and `graphlib.TopologicalSorter` are all I could really see us using in this package... but I already do use `removesuffix` and builtin generics in `gen_masks`, and anticipate using `TopologicalSorter` if we do ever make our own task scheduler - it would do most of the hard work for us.

https://endoflife.date/python

Reply:
I removed PyPy because there is some conflict with `scipy` that makes installation fail... could probably look into this in re-enable it.


https://github.com/brain-microstructure-exploration-tools/abcd-microstructure-pipelines/actions/runs/8442097656/job/23122871135

Reply:
Agreed with python 3.9 as the min -- I just meant to ask about matrix include line.

I don't know anything about pypy so no complaints :)